### ISSUE TITLE
North-East of France doesn't work

### ISSUE THREAD

**Issue by Altonss** [grote/Transportr#779]
**Describe the bug**
The North-East of France network doesn't work. I'm unable to search and find any station.

**To Reproduce**
1. Go to 'Search'
2. Type any known station
3. Nothing happens

**Expected behavior**
A result should appear, with several suggestion of stations.

**Versions (please complete the following information):**
 - **Transportr Version:** 2.1.3
 - **Device:** Samsung Galaxy Core Prime
 - **Android Version:** Android 5.0.1

**Comment by julianfairfax** [comment]
I'm having this same problem in Switzerland

**Comment by ialokim** [comment]
Thanks for your report! Are you still experiencing the issue? If so, which stations were you trying?

Most probably the bug was due to an error at the data source which has been fixed meanwhile.

**Comment by Altonss** [comment]
> Thanks for your report! Are you still experiencing the issue? If so, which stations were you trying?

Yes I am still experiencing the issue, all the stations in North-East of France (for example "Lille Flandres"). I can try to search anything, there is no suggestion or response to the query.



> Most probably the bug was due to an error at the data source which has been fixed meanwhile.

Yes that's what I understood too. It seems to be an upstream issue with Navitia (see https://github.com/schildbach/public-transport-enabler/pull/402 and maybe https://github.com/schildbach/public-transport-enabler/issues/304)

**Comment by ialokim** [comment]
> "Lille Flandres"

I've just tried it and it shows several suggestions, including "Gare Lille Flandres", on my phone. This looks like it might be indeed related to #781 then. Although we experienced this problem before only for Android versions up to 4.4: #721

**Comment by Altonss** [comment]
Oh that is great news thank you very much, the suggestion "Gare Lille Flandres" you mentionned is correct! 
I already wrote in #781 but yes #721 may be related. As I mentionned in the first message, I am using Android 5.0.1, so this might be the issue here :) I hope we will soon find a solution!
